scripts/on-premises/gcp/gce_image_builder.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
```

Comment:
Please create rather a Golang command under https://github.com/google/android-cuttlefish/tree/main/tools/baseimage/cmd.

The original cuttlefish script to create host images was written in bash and had to be rewritten to Golang https://cs.android.com/android/platform/superproject/main/+/main:device/google/cuttlefish/tools/create_base_image.go. 

Existing command https://github.com/google/android-cuttlefish/blob/main/tools/baseimage/cmd/create_gce_x86_64_image/main.go has the GCE API calls you need to create the image.